texts/plain/CAL-1.0

```diff
@@ -0,0 +1,137 @@
+The Cryptographic Autonomy License, v. 1.0
+
+This Cryptographic Autonomy License (the �License�) applies to any Work whose owner has marked it with any of the following notices, or a similar demonstration of intent:
+
+SPDX-License-Identifier: CAL-1.0
+Licensed under the Cryptographic Autonomy License version 1.0
+
+or
+
+SPDX-License-Identifier: CAL-1.0-Combined-Work-Exception
+Licensed under the Cryptographic Autonomy License version 1.0, with Combined Work Exceptio
+
+______________________________________________________________________
+
+1. Purpose
+This License gives You unlimited permission to use and modify the software to which it applies (the �Work�), either as-is or in modified form, for Your private purposes, while protecting the owners and contributors to the software from liability.
```

Comment:
the artifacts here make me think there's an issue with the encoding of the file. Am I wrong?

Reply:
Fixed the typo in the PR and changed the encoding to UTF-8 which should resolve the issues.